### PR TITLE
Allow `jvm_import` to accept a `jar` parameter

### PR DESCRIPTION
**Pull request by shs96c**
We only allow a single `srcjar`, and the `jars` parameter is already checked to ensure the length is 1.

This also deprecates the original `jars` parameter.

### CODE REVIEW

File: private/rules/jvm_import.bzl [hunk 1]
```diff
@@ -55,6 +66,10 @@ def _jvm_import_impl(ctx):
         progress_message = "Creating compile jar for %s" % ctx.label,
     )
 
+    additional_providers = []
```
Comment (jin) [inline private/rules/jvm_import.bzl:69]: unrelated?

File: private/rules/jvm_import.bzl [hunk 2]
```diff
@@ -91,6 +109,12 @@ jvm_import = rule(
         "neverlink": attr.bool(
             default = False,
         ),
+        "maven_coordinates": attr.string(
```
Comment (jin) [inline private/rules/jvm_import.bzl:112]: unrelated?

**Review by jin — APPROVED** [review]

File: private/rules/jvm_import.bzl (hunk 1 above)
Comment (shs96c) [reply private/rules/jvm_import.bzl:69]: Yes. Removed.

File: private/rules/jvm_import.bzl (hunk 2 above)
Comment (shs96c) [reply private/rules/jvm_import.bzl:112]: Yes. Removed.

**Review by jin — APPROVED** [review]
